src/tests/oracle.rs

```diff
@@ -1607,6 +1609,125 @@ fn get_data_before() {
 	});
 }
 
+#[test]
```

Comment:
These two tests relate to governance functionality, so could they be moved to `tests/governance.rs` please?